Classes/View/LegacyPreviewView.php

```diff
@@ -54,17 +54,17 @@ class LegacyPreviewView extends PreviewView {
     );
 
 
-    /**
+	/**
 	 * @codeCoverageIgnore
-     * @param array $row
-     * @param Column $column
-     * @param $colPosFluxContent
-     * @param $dblist
-     * @param $target
-     * @param $id
-     * @param $content
-     * @return string
-     */
+	 * @param array $row
+	 * @param Column $column
+	 * @param $colPosFluxContent
```

Comment:
While you are doing CGL anyway: please add parameter types for this and the next handful of arguments.